IncludedFileInfo.cpp

```diff
@@ -113,22 +113,21 @@ class FuncDeclVisitor : public clang::RecursiveASTVisitor<FuncDeclVisitor> {
       case clang::BuiltinType::Kind::Char_U:
```

Comment:
I think all the case statements for various interger Kinds can be replaced by the test `BltInTy->isInteger()`.

Reply:
I've applied this, thanks!